tools/distpartitioning/data_shuffle.py

```diff
@@ -576,7 +584,7 @@ def gen_dist_partitions(rank, world_size, params):
     print('[Rank: ', rank, '] Done assigning global_ids to edges ...')
 
     #determine global-ids for edge end-points
-    get_shuffle_global_nids_edges(rank, world_size, edge_data, node_part_ids, node_data)
```

Comment:
the original implementation change `edge_data`?

Reply:
Yes. Originally this function call changed the edge_data dictionary (added SHUFFLE_GLOBAL_SRC_ID and SHUFFLE_GLOBAL_DST_ID keys to it). But somehow this was missed when we made a RULE that if a function changes a variable it should `return` the said variable. 

So now I modified this function call to maintain the consistency.